src/app/frontend/index_config.js

```diff
@@ -17,13 +17,23 @@
  * @ngInject
  */
 export default function config($mdThemingProvider) {
+  let kubeBlue = '326de6';
```

Comment:
I see that the palette configuration object is the same. How about extracting it here to a common variable, instead of just the kubeBlue color?